### ISSUE TITLE
Diacritic dot support for Italics

### ISSUE THREAD

**Issue by Elendil03** [adobe-fonts/source-serif-pro#61]
Hey guys,
I'd love to see the implementation of glyphs containing the diacritic dots also for the Italics of SSP.
Is there any chance of doing that? I don't think it should be too complicated as the base glyphs already exist. Especially the ḥ glyph is very important to me. It's such a pity that this amazing typeface cannot be used in works containing DMG transcriptions for that reason.
Regards,
- Elendil 03

BTW, the italics done so far look excellent!

**Comment by frankrolf** [comment]
The “diacritic point” – which exact sequence of code points are you thinking about?
There’s a large chance the font already supports it – if there is a discrepancy between Roman and Italics it is no longer deliberate (as of release 3.000, issued last week):
https://github.com/adobe-fonts/source-serif-pro/releases/tag/3.000R

**Comment by Elendil03** [comment]
I noticed that in English, it's called a diacritic dot, not point (Ja, das Deutsche kann manchmal ziemlich verwirrend sein). Anyway, it's not included in ISO 8859, so I doubted it was already done. These signs only exist in Unicode, I think. However, they actually are included in the updated Italics. That's wonderful, huge thanks to you.

**Comment by frankrolf** [comment]
I assume this was about the _Umlaut_, two consecutive dots also known as _diaeresis_ or _trema_.
Yes, U+0308 COMBINING DIAERESIS is supported across all the styles, and is wired up in the mark feature.
I think it should be available in older font versions as well.

**Comment by Elendil03** [comment]
I was referring to [this ](https://en.wikipedia.org/wiki/Dot_(diacritic)) single diacritic element (◌̇ or ◌̣), not the trema (¨).

**Comment by Elendil03** [comment]
OK, so after getting a display error (text rendered to lightly on small sizes) and reinstalling LibreOffice, the text processor I'm using, the  glyph ḥ (h + ◌̣)  is again displayed wrongly (it turns into a sans-serif _printer's pie_, if you can trust dict.cc, in German _Zwiebelfisch_), but only when set on Italics, the Roman is alright. Any idea on how to solve this annoying issue?

**Comment by frankrolf** [comment]
Sorry for misunderstanding – it is easier to mention the code point right from the beginning.
I am trying to reproduce the error in different text engines, but don’t encounter a problem.

I am using the following combinations:
`h` + `U+0307 COMBINING DOT ABOVE`
`h` + `U+0323 COMBINING DOT BELOW`

_CoreText (Apple Safari)_
<img width="952" alt="Screen Shot 2019-06-19 at 16 09 23" src="https://user-images.githubusercontent.com/2049645/59772942-01e08680-92ad-11e9-87f4-c8912f57bd37.png">

_FreeType (Google Chrome)_
<img width="952" alt="Screen Shot 2019-06-19 at 16 07 22" src="https://user-images.githubusercontent.com/2049645/59773090-4ec45d00-92ad-11e9-9896-0291fef272cd.png">

_All possible combinations involving the COMBINING DOTs as rendered in Adobe InDesign_
![2019-06-19_161608](https://user-images.githubusercontent.com/2049645/59773246-9b0f9d00-92ad-11e9-8e31-1d31ccc88301.png)

(see also  [2019-06-19_160747.pdf](https://github.com/adobe-fonts/source-serif-pro/files/3306499/2019-06-19_160747.pdf) )

In short, I cannot reproduce the issue.

I just installed LibreOffice for Mac to try and reproduce the problem – it seems that an older version of Source Serif is bundled with the application. Since U+1E25 LATIN SMALL LETTER H WITH DOT BELOW is part of Adobe Latin-4 (which is the new Latin character set since the 3.0 update), it is possible that that mark-to-base combination does not exist in a previous version of Source Serif either.
 
Unfortunately, this list does not mention the version number: https://wiki.documentfoundation.org/Fonts

Please let me know if you find out how to deactivate bundled fonts in LibreOffice, and if you manage to use Source Serif 3.0 in it.

**Comment by Elendil03** [comment]
Well, it seems that when running Windows, the only current solution is installing the font **after** LibreOffice and every upcoming update up until the inclusion of Source Serif Pro 3.0. With the overwritten font, however, I get a very weird rendering with OpenGL turned off (as my laptop's video card seems to be on the OpenGL "blacklist"). This for good reason: the letters appear really unclean when I override this blacklisting through the options:
[img]https://i.ibb.co/pyb9Wx9/d.jpg[/img]
[img]https://i.ibb.co/WpQNG7Z/a.jpg[/img]
On the other hand, with it turned off, the rendering becomes a true eyesore when zoomed out
[img]https://i.ibb.co/QkVNScg/b.jpg[/img]
Zoomed in, on the other hand, it all looks fine.
[img]https://i.ibb.co/9nBkYg6/c.jpg[/img]
I prefer the solution with OpenGL despite the ruggedness, but only as a compromise solution.
The older font version included in LO, though, works fine when rendered without OpenGL.

The good thing is that, apparently, the LO-bundled fonts are installed to the default Windows location so that they can still be removed or modified.

What are your thougts on this rendering bug?

**Comment by frankrolf** [comment]
I suggest opening a new issue for this rendering problem – however I assume there’s not much I can do on my end. When you open the new bug, please specify which outline version you use (OTF or TTF).

FYI: To make the images show up in-line, I inserted them via drag-and-drop.

**Comment by Elendil03** [comment]
[Done](https://github.com/adobe-fonts/source-serif-pro/issues/62). Thanks again for your effort.

**Comment by fitojb** [comment]
For the record, LibreOffice [is now updated](https://cgit.freedesktop.org/libreoffice/core/commit/?id=e045fcca8699c745a2788bc73ba9b5946734eb17) to bundle version 3 of the Source Serif fonts.

**Comment by frankrolf** [comment]
Great news @fitojb! Thank you!

**Comment by Elendil03** [comment]
They were actually faster than what I'd expected. I mean, they gotta have a lot of bundled fonts to check for updates.
